controllers/topics.controllers.js

```diff
@@ -2,8 +2,11 @@ const { showTopics } = require('../models/topics.models')
 
 
 exports.getTopics = (req, res, next) => {
-    showTopics().then( (topics) => {
+    showTopics()
+    .then( (topics) => {
         res.status(200).send( topics )
```

Comment:
Does this work? Shouldn't the .send() have an object inside it? such as `{ topics: topics }`

Reply:
It's working, as it is sending an array.